vortex-serde/src/io/tokio.rs

```diff
@@ -46,6 +34,25 @@ impl<W: AsyncWrite + Unpin> VortexWrite for TokioAdapter<W> {
     }
 }
 
+impl VortexRead for File {
```

Comment:
this is not correct. You can just wrap a file in a cursor and it will have vortexread impl

Reply:
oh it is correct, fine I read wrong